File: njnu-classroom-mybatis/src/main/kotlin/cn/repigeons/njnu/classroom/mybatis/service/NoticeService.java
package cn.repigeons.njnu.classroom.mybatis.service;


import cn.repigeons.njnu.classroom.mybatis.mapper.NoticeMapper;
import cn.repigeons.njnu.classroom.mybatis.model.Notice;
import com.mybatisflex.spring.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 公告记录 服务层实现。
 *
 * @author mybatis-flex-helper automatic generation
 * @since 1.0
 */
@Service
public class NoticeService extends ServiceImpl<NoticeMapper, Notice> implements INoticeService {

}

File: njnu-classroom-portal/src/main/kotlin/cn/repigeons/njnu/classroom/service/NoticeService.kt
package cn.repigeons.njnu.classroom.service

import cn.repigeons.njnu.classroom.model.vo.NoticeVO

interface NoticeService {
    fun putNotice(text: String): NoticeVO
    fun getNotice(): NoticeVO?
    fun setNotice(id: Int): NoticeVO?
}